src/webrtc_session.ts

```diff
@@ -114,7 +112,7 @@ export class WebRTCSession extends BaseSession {
         this.pc.onconnectionstatechange = () => {
             const state = this.pc.connectionState
             console.log("new connection state", state, marker)
-            if ((state == "connected") && (marker != null)) {
+            if ((state === "connected") && (marker != null)) {
```

Comment:
I Prefer to use  '===' & '!==' only is cases where they're clearly helpful, like in `a === "undefined"`.
In this case the value comes from the browser and it's always a string so we can keep the code simpler and save a char!